packages/babel-generator/test/fixtures/flow/interfaces-module-and-script/expected.js

```diff
@@ -1,9 +1,23 @@
-interface A {};
-interface A extends B {};
-interface A<T> extends B<T>, C<T> {};
-interface A { foo: () => number };
-interface Dictionary { length: number, [index: string]: string, };
+interface A {}
+;
```

Comment:
Looking at the AST from Flow itself, the semicolons are not in the grammar for interface declarations, which means the current behavior of parsing them as EmptyStatement nodes is correct. We should probably just remove the semicolons from the `actual.js` files, but I understand if you don't want to for now.